README

```diff
@@ -18,7 +18,7 @@ Installation for FreeBSD, Linux and OpenBSD:
 Installation for OS X:
 ----------------------
 1. Create a subdirectory rogue/ in your quake2 directory.
-2. Copy pak0.pak and videos/ from the the Reckoning CD to
```

Comment:
this should be `the The Reckoning CD` (or maybe `the "The Reckoning" CD) - as "The Reckoning" is the addons name (same below).